OsuPlayer/Windows/ExportSongsProcessWindow.axaml.cs

```diff
@@ -74,29 +76,68 @@ internal async Task ExportSongs(CancellationToken token)
 
                 var hashLength = mapEntry.Hash.Length < 8 ? mapEntry.Hash.Length : 8;
 
-                var fileName = $"{mapEntry.GetArtist()} - {mapEntry.GetTitle()} ({mapEntry.Hash.Substring(0, hashLength)}).mp3";
```

Comment:
Is there are reason why you use the properties now instead of the implementation methods that support unicode aswell?

Reply:
Yes because I read the BASS audio transcoder does some weird things using anything other than ANSI in the path

Reply:
Also some audio players (e.g. in cars) have problems playing files with unicode symbols

Reply:
Ahh I see, fair point. What do you think then, if we change the Interface for `GetArtist()` and `GetTitle()` that it accepts a `bool` so users can toggle it as well, and the default would be false of course?

What do you think of that idea?